charts/argo-cd/templates/argocd-server/certificate.yaml

```diff
@@ -31,6 +31,10 @@ spec:
     {{- end }}
     kind: {{ .Values.server.certificate.issuer.kind | quote }}
     name: {{ .Values.server.certificate.issuer.name | quote }}
+  privateKey:
```

Comment:
Please check for the whole specification of the privateKey below and use following construct:

```yaml
{{- with .Values.server.certificate.privateKey }}
privateKey:
  {{- toYaml . | nindent 4 }}
{{- end }}
```

Reply:
I've updated to wrap all of the `privateKey` as suggested.